src/App.less

```diff
@@ -1,5 +1,7 @@
+@import "~nav-frontend-core/less/core";
```

Comment:
Husker ikke helt om det er nødvendig å dra inn `core` på denne måten, eller om det ordnes med en gang man bruker en komponent fra nav-frontend. Om det er nødvendig så se bort fra denne kommentaren. ;) 

Men om denne brukes for at `@navLysGra` skal være definert så ville jeg endret til `@import '~nav-frontend-core/less/_variabler';` eller `@import (reference) '~nav-frontend-core/less/core'`
Begge deler gjør at man bare får variabler og mixins, men ingen less/css blir skrevet til fil. På den måten unngår vi at `core` blir duplisert når vi bygger det sammen.